app/api.go

```diff
@@ -175,6 +176,36 @@ func PrepareApp(env env.Project, options *PrepareOptions) (err error) {
 
 					if metadata.Shim == "plugin" {
 						//look for Makefile and execute it
+						makeFilePath := path.Join(env.GetVendorSrcDir(), value.Ref, dirShim, makeFile)
+						fmt.Println("Make File:", makeFilePath)
+						fgutil.CopyFile(makeFilePath, path.Join(cmdPath, makeFile))
+
+						// Copy the vendor folder (Ugly workaround, this will go once our app is golang structure compliant)
```

Comment:
Might it be better to just let the makefile copy the vendor dir for now, rather than the CLI? Put the burden on the plugin style shims for now?

Reply:
The problem is that the Makefile is not aware of the root path and adds a bit of complexity to the Makefile for a very temporary workaround, push error handling and cleanup in Makefiles are tough, so I believe in the cli even if it is ugly it is the best spot.